pkg/report/guilty.go

```diff
@@ -52,3 +56,24 @@ nextFile:
 	}
 	return ""
 }
+
+func GetMaintainers(linux string, file string) ([]string, error) {
+	// ./scripts/get_maintainer.pl is a Linux kernel script.
+	output, err := osutil.RunCmd(time.Minute, linux, "./scripts/get_maintainer.pl", "-f", file)
+	if err != nil {
+		return nil, fmt.Errorf("get_maintainer.pl failed: %v\n%v", err, string(output))
+	}
+	parts := strings.Split(string(output), "\n")
+	var mtrs []string
+	for _, part := range parts {
+		if !strings.Contains(part, "maintainer") && !strings.Contains(part, "supporter") &&
```

Comment:
There are flags for this in get_maintainer.pl. It will be more reliable if we fetch only emails and only necessary emails. It also will make it simple to explain how we obtain list of maintainers to kernel developers. And won't break when output changes.